source/app/service-providers/translations/index.ts

```diff
@@ -105,6 +105,20 @@ export default class TranslationProvider extends ProviderContract {
     global.i18nFallbackRawData = await fs.readFile(fallback.path, 'utf8')
     global.i18nFallback = JSON.parse(global.i18nFallbackRawData)
 
+    // Get user's setting of translation updates
+    const checkForTranslationUpdates: boolean = this._config.get('system.checkForTranslationUpdates')
+    if (checkForTranslationUpdates) {
+      await this.update()
```

Comment:
An additional thing I just noticed: Instead of awaiting this promise here, we could also write `this.update().catch(err => logTheError)`.

The reason is that this will speed up the boot process, because the whole app does not have to wait for translations to download. Any changes are only applied after a restart anyway. This could make the app snappier, especially when users have slow internet connections.

We could then simply log an error if something went wrong.

Reply:
> An additional thing I just noticed: Instead of awaiting this promise here, we could also write `this.update().catch(err => logTheError)`.
> 
> The reason is that this will speed up the boot process, because the whole app does not have to wait for translations to download. Any changes are only applied after a restart anyway. This could make the app snappier, especially when users have slow internet connections.
> 
> We could then simply log an error if something went wrong.

Changed both. Is that look good to you?